.env.example

```
@@ -146,3 +146,11 @@ WEB_INTEL_SELF_URL=http://localhost:4002
 WEB_INTEL_V2_SELF_URL=http://localhost:4003
 ANALYSIS_AGENT_SELF_URL=http://localhost:4004
 REPORT_AGENT_SELF_URL=http://localhost:4005
+
+# Cluster Runtime Environments
+PORT=3000
+REGISTRY_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The REGISTRY_URL key is duplicated

(DuplicatedKey)